### PR TITLE
Fix add to cart

### PR DESCRIPTION
**Pull request by LisaElaine**
gave access to devise views, 
update modal for sign in, 
created a popover and then removed it because we should not have it for a mobile view

### CODE REVIEW

File: app/views/users/confirmations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (jessiebbaxter) [inline app/views/users/confirmations/new.html.erb]: Not following what this page is for?

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+
+<div class="wrapper">
+  <div class="container">
+    <h1>Your Cart</h1>
+    <!-- <h3>Order number #<%= @order.id %></h3> -->
+    <table class="table">
+      <thead>
+        <tr>
+          <th style="width: 50%", scope="column" colspan="2">Product</th>
```
[the remark this answers is not in the document]
Comment (LisaElaine) [reply app/views/orders/show.html.erb]: Caroline's code :)

File: app/views/products/show.html.erb [hunk 3]
```diff
@@ -19,18 +19,19 @@
...
 					<div class="product-actions">
-		              <% if current_user %>
-		                <div class="action-add">
-		                  <%= link_to "Add to Cart", product_order_items_path(@product), method: :post, class: "btn btn-primary" %>
-		                </div>
-		               <% else %>
-		               <!-- Modal link -->
-		                <%= link_to "Add to Cart", "#", class: "btn btn-primary", :id => 'add-product', data: {toggle: "modal", target: "#signInModal"}  %>
-		                <% end %>
+          <% if current_user %>
```
[the remark this answers is not in the document]
Comment (LisaElaine) [reply app/views/products/show.html.erb]: Updated some of the code, happy for the indentation to be changed. :)

File: app/views/shared/_navbar_logged_in.html.erb [hunk 4]
```diff
@@ -7,6 +7,7 @@
 
     <a href="/" class="navbar-wagon-item navbar-wagon-link">Dashboard</a>
     <a href="#" class="navbar-wagon-item navbar-wagon-link">Add a product</a>
+    <%= link_to '<i class="fas fa-shopping-cart"></i>'.html_safe, order_path(current_user.find_or_create_pending_order) %>
```
[the remark this answers is not in the document]
Comment (LisaElaine) [reply app/views/shared/_navbar_logged_in.html.erb]: app/models/user.rb

**Pull request merged**